cmd/agent/gui/views/templates/generalStatus.tmpl

```diff
@@ -520,4 +523,4 @@
       {{ end }}
     </span>
   </div>
-{{- end -}}
+{{- end -}}
```

Comment:
NIT: I think we can revert this. most of our files have a trailing newline IIRC (However I don't think this actually hurts anything)